assets/product-compare.scss.liquid

```diff
@@ -0,0 +1,125 @@
+.compare-blender {
+.product-comparison--bg-image{
+    background-position: 0 50%;
+    background-size: cover;
+    display: inline-block;
+    width: 100%;
+}
+.product-comparison--header-container {
+    padding: 100px 0 120px;
+}
+.product-comparison--sub-leader {
+    font-size: 28px;
+    font-size: 2rem;
+    color: #fff;
+    text-align: center;
+    text-transform: uppercase;
+}
+.product-comparison--leader {
+    font-size: 56px;
+    font-size: 4rem;
+    color: #fff;
+    text-align: center;
+    text-transform: uppercase;
+}
+  #thePopup{
+    width:60%;
+    left: 20%;
+  }
+
+@media screen  and (max-width: 840px) /* Tablet */ {
+  #thePopup{
+    width:80%;
+    left: 10%;
+  }
+
+}
+
```

Comment:
rather than putting our own breakpoints in let's use the Shopify $small $medium $large breakpoints so our elements react in concern with the rest of the theme.